roles/en-scn-init/tasks/init_scn.yml

```diff
@@ -29,7 +29,7 @@
 
 - name: "Generate SCN data & genesis.json by Homi"
   become: yes
-  command: homi setup --gen-type local {{ homi_default_options }} {{ homi_extra_options }} --test-num {{ homi_test_account_num }} --cn-num {{ groups.scn|length }} --chainID {{ kaia_service_chain_id }} --network-id {{ kaia_service_network_id }} --p2p-port {{ kaia_conf.PORT }} -o {{ homi_results_dir }}
+  command: homi setup --gen-type local {{ homi_default_options }} {{ vars['homi_extra_options_' + kaia_node_type] | default('') }} --test-num {{ homi_test_account_num }} --cn-num {{ groups.scn|length }} --chainID {{ kaia_service_chain_id }} --network-id {{ kaia_service_network_id }} --p2p-port {{ kaia_conf.PORT }} -o {{ homi_results_dir }}
```

Comment:
If this PR allow to specify mnemonic per **node-type(cn/pn/en)**, `homi` generates different `genesis.json`. Should we also copy one genesis.json as the only one?

* https://github.com/kaiachain/kaiaspray/blob/06f50029fae89e5c8d93539684cf8916faaa0594/roles/node-init/tasks/copy_files.yml#L56-L64

Reply:
I tried before but I didn't apply it. Why I didn't apply it is because kaiaspray needs EN's kni for setup the servicechain bridge so en's kni needs to be generated also. It doesn't generated by specifying `en-num`.
What I was thinking is fixing the homi to generate the EN's kni when en-num is specified. Then, we can consider specifying `en-num`.

Reply:
Thanks for your confirmation!